Cubical/Data/Int/Properties.agda

```diff
@@ -117,14 +169,74 @@ _+negsuc_ : Int → ℕ → Int
 z +negsuc 0 = predInt z
 z +negsuc (suc n) = predInt (z +negsuc n)
 
+infix  8 -_
+infixl 7 _·_ _⊓_
+infixl 6 _+_ _-_ _⊔_
+infix  4 _<_ -- _≤_
+
 _+_ : Int → Int → Int
 m + pos n = m +pos n
 m + negsuc n = m +negsuc n
 
+-- TODO: in `Cubical.Papers.RepresentationIndependence` we have
+--   -_ : ℤ → ℤ
+--   - x = 0 - x
+-- there might be an advantage of defining -_ by _-_ and not the other way around
+-- e.g. in Cubical.HITs.QuoInt we have
+--   -_ : ℤ → ℤ
+--   - signed s n = signed (not s) n
+--   - posneg i   = posneg (~ i)
+
+-_ : Int → Int
+- pos  zero   = pos zero
+- pos (suc n) = negsuc n
+- negsuc   n  = pos (suc n)
+
 _-_ : Int → Int → Int
-m - pos zero    = m
-m - pos (suc n) = m + negsuc n
-m - negsuc n    = m + pos (suc n)
+m - n = m + (- n)
+
+_·_ : Int → Int → Int
+x · y  = signed (sgn x ⊕ sgn y) (abs x ·ⁿ abs y)
+
+_<_ : ∀(x y : Int) → Type₀
+pos    x < pos    y = x <ⁿ y
+pos    x < negsuc y = ⊥
+negsuc x < pos    y = ⊤
+negsuc x < negsuc y = y <ⁿ x
```

Comment:
definition in terms of `<` on `Cubical.Data.Nat`

Reply:
This is not necessary for the commring-strucure, of course, but I would like to push some order-related properties if that is okay (?)